packages/api/src/platforms/vtex/resolvers/validateCart.ts

```diff
@@ -110,19 +110,27 @@ const equals = (storeOrder: IStoreOrder, orderForm: OrderForm) => {
 }
 
 function hasChildItem(items: OrderFormItem[], itemId: string) {
-  return items?.some(item => item.parentItemIndex && items[item.parentItemIndex].id === itemId)
```

Comment:
**question**: The bug here is because when  **parentItemIndex** is 0, the function is returning false?

Reply:
Yes! Basically the function always returned false for the first item